winex.js

```diff
@@ -116,7 +114,7 @@ stripNulls = function(obj) {
   for (k in obj) {
     if (!hasProp.call(obj, k)) continue;
     v = obj[k];
-    if (v === null) {
+    if (typeof v === 'undefined' || v === null) {
```

Comment:
This was the only actual code change needed to keep the tests passing after replacing deep-extend with Object.assign. The version of deep-extend that was being used appears to have been filtering undefined properties itself (though it looks like newer versions of it don't, not that it matters since we're dropping it).